packages/foundation-models/package.json

```
@@ -33,6 +33,7 @@
     "@sap-ai-sdk/ai-api": "workspace:^",
     "@sap-ai-sdk/core": "workspace:^",
     "@sap-cloud-sdk/http-client": "^3.24.0",
-    "@sap-cloud-sdk/util": "^3.24.0"
+    "@sap-cloud-sdk/util": "^3.24.0",
+    "@sap-cloud-sdk/connectivity": "3.24.0"
```

```suggestion
    "@sap-cloud-sdk/connectivity": "^3.24.0"
```
[req] Please consider this in other packages as well

---

Thanks for pointing these out. I also found some before. Missed them and trusted autocompletion.